### ISSUE TITLE
Extreme lag + no cursor

### ISSUE THREAD

**Issue by gwa100** [MindShow/USBDisplay#55]
I am using version 2.0.1.7.3 of the software on Windows 11 22H2, everything works fine but when do anything on the usb connected display it is really laggy and when games are played it is basically at 1 FPS, Also there is no mouse cursor!

[WinUsbDisplay.log](https://github.com/MindShow/USBDisplay/files/10609857/WinUsbDisplay.log)
Heres my log file if it helps!

**Comment by furiiibond** [comment]
same

**Comment by AbueloRaul** [comment]
Did you manage to solve it? that happens to me too

**Comment by Agetoblack** [comment]
It is better to return the adapter. I've been waiting for almost a year for the driver developers to fix something, but they don't do anything.

**Comment by AbueloRaul** [comment]
what adapter you recomend me ?

**Comment by Agetoblack** [comment]
No lo he probado porque es caro, pero he leído que este funciona. Los que tienen "DisplayLink" certificado deberían funcionar. Si lo compras y te sirve sería bueno que nos avisaras, si no te sirve también puedes regresarlo al vendedor.

https://a.co/d/cLxi6Jz

**Comment by AbueloRaul** [comment]
jaja ojala en ecuador se puedan regresar las cosas cuando las compras, pero le probare, justo hoy dia seguramente voy a averiguar sobre estos adaptadores y compare uno ! esperemos encontrar es a ver que tal !

**Comment by Alespost** [comment]
You have to set app to use internal GPU - worked for me

**Comment by Agetoblack** [comment]
How you did it? Because I change it to the internal GPU and it only shows the content but with lag and no cursor, before that it didn't show anything, just black screen.

**Comment by AbueloRaul** [comment]
@Alespost  how to change to the internal cpu?

**Comment by Agetoblack** [comment]
Yes, I want to see if I did something wrong and make it work for me too

**Comment by Alespost** [comment]
Start  > Settings  > System > Display > Graphics Settings and then add "Windows USB Display" app with energy saving option.

There is possibility you have CPU without internal graphics card so you only have external one. In that case you won't be able to do this change

**Comment by gwa100** [comment]
I already done that, i also tried disabling my discrete GPU, also did not help

**Comment by mosioc** [comment]
I have exactly the same problem on my laptop (Windows 11). When I connect only 1 external monitor via usb to hdmi adapter, there is no problem, the cursor is fine and there is no lag.
But as soon as I connect another monitor to the default hdmi port, the cursor disappears from the previous monitor and lag occurs. It should be mentioned that I have set the "Windows USB Display" to run on integrated GPU.

**Comment by gwa100** [comment]
i found a fix, if you have iGPU and dGPU, disable your iGPU and if on laptop disable your laptop screen aswell, i did this and the cursor appears with a lot less lag than what it was before, works with the newest version aswell, but no portrait mode sadly

**Comment by MindShow** [comment]
new version driver，
support multi GPUs
support win11
support portrait mode
[USMDisplay_Windows 3.0.1.40](http://www.ultrasemi.com/usbdisplay/Windows/USMDisplay_Windows.exe)
